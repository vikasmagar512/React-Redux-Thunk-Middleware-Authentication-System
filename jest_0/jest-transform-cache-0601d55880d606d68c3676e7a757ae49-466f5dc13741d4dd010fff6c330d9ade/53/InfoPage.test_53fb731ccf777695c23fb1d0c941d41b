'use strict';
var _react = require('react');var _react2 = _interopRequireDefault(_react);
var _createReactClass = require('create-react-class');var _createReactClass2 = _interopRequireDefault(_createReactClass);
var _propTypes = require('prop-types');var _propTypes2 = _interopRequireDefault(_propTypes);
var _reactRouter = require('react-router');
var _reactDom = require('react-dom');var _reactDom2 = _interopRequireDefault(_reactDom);
var _reactTestRenderer = require('react-test-renderer');var _reactTestRenderer2 = _interopRequireDefault(_reactTestRenderer);
var _enzymeToJson = require('enzyme-to-json');var _enzymeToJson2 = _interopRequireDefault(_enzymeToJson);
var _enzyme = require('enzyme');
var _InfoPage = require('../../../src/components/info/InfoPage');var _InfoPage2 = _interopRequireDefault(_InfoPage);function _interopRequireDefault(obj) {return obj && obj.__esModule ? obj : { default: obj };}


describe('InfoPage', function () {
  it('should match its empty snapshot', function () {
    var tree = _reactTestRenderer2.default.create(
    _react2.default.createElement(_InfoPage2.default, null)).
    toJSON();
    expect(tree).toMatchSnapshot();
  });
  /*    it('renders without crashing', () => {
              const div = document.createElement('div');
              ReactDOM.render(<App/>, div);
          });
      */
});