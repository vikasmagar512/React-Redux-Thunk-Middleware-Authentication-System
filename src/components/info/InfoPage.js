import React from 'react';

class InfoPage extends React.Component {
  render() {
    return (
      <div>
        <h1>Info</h1>
        <p>coming soon!</p>
      </div>
    );
  }
}

export default InfoPage;