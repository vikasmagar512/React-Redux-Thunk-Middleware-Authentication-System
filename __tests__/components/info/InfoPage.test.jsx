
import React,{Component} from 'react';
import createReactClass from  'create-react-class';
import PropTypes from 'prop-types'
import { Link, IndexLink } from 'react-router';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import InfoPage from '../../../src/components/info/InfoPage';


describe('InfoPage', () => {
  it('should match its empty snapshot', () => {
    const tree = renderer.create(
      <InfoPage/>
     ).toJSON();
    expect(tree).toMatchSnapshot();
  });
/*    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<App/>, div);
    });
*/   
}); 