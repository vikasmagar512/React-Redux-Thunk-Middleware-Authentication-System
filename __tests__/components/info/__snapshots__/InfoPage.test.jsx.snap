// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoPage should match its empty snapshot 1`] = `
<div>
  <h1>
    Info
  </h1>
  <p>
    coming soon!
  </p>
</div>
`;
